crates/hdk/src/entry.rs

```diff
@@ -129,8 +139,13 @@ where
 /// means for the happ.
 ///
 /// ```ignore
-/// #[hdk_entry(id = "foo")]
-/// struct Foo(u32);
+/// #[hdk_entry_helper]
+/// pub struct Foo(u32);
+///
+/// #[hdk_entry_defs]
+/// pub enum EntryTypes {
+///    Foo(Foo)
+/// }
 ///
 /// let foo_zero_action_hash: ActionHash = commit_entry!(Foo(0))?;
```

Comment:
I think commit_entry! is actually outdated, which I didn't catch the first time around.

```suggestion
/// let foo_zero_action_hash: ActionHash = create_entry(EntryTypes::Foo(Foo(0)))?;
```